### PR TITLE
refactored using async/await

### PR DESCRIPTION
**Pull request by fduarte42**
To prepare for direct puppeteer api and for more readable code, I refactored the code using async/await.

### CODE REVIEW

**Comment by fduarte42** [comment]
The Tests have to be adapted for async/await

**Comment by spiritix** [comment]
The problem is that async/await is only supported by Node >= 7.6. I am not sure how many applications are still using Node 6.

**Comment by fduarte42** [comment]
Perhaps we need a build system to transpile to node6

**Comment by spiritix** [comment]
Feel free to add a build system, however I currently want to keep support for Node 6.

**Comment by fduarte42** [comment]
I added babel-node6-es6 present transpiler. You can call it with "npm run build". The Converter.js in lib will be created automatically. Source is in lib-src.

File: index.js [hunk 1]
```diff
@@ -12,18 +12,16 @@ const getStdin = require('get-stdin');
 const program = require('commander');
```
Comment (spiritix) [inline index.js]: what about index.js - wouldn't it need to be transpiled as well?

**Pull request closed without merging**